packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItemsGroup.tsx

```diff
@@ -80,7 +77,7 @@ export const AccountItemsGroup = ({
             />
             {showStaking && (
                 <Wrapper>
-                    <Divider $selected={selected} $isBigger />
+                    <Divider $isBigger />
```

Comment:
[optional] We have a `Divider` component in our design system. 

Its pretty simple now (`components/src/components/Divider/Divider.tsx`) but maybe this is a good oportunity to extend it a use it here. 

@jvaclavik WDYT?

Reply:
(actually, it might make sense to rename it to something else)